modules/cluster-logging-upgrading-logging.adoc

```diff
@@ -5,98 +5,135 @@
 [id="cluster-logging-updating-logging_{context}"]
 = Updating cluster logging
 
-After updating the {product-title} cluster, you can update cluster logging from 4.2 to 4.3 by updating the subscription for the Elasticsearch Operator and the Cluster Logging Operator.
+[role="_abstract"]
+After updating the {product-title} cluster, you can update cluster logging from 4.4 to 4.6 by changing the subscription for the Elasticsearch Operator and the Cluster Logging Operator.
+
+When you update:
+
+* You must update the Elasticsearch Operator before updating the Cluster Logging Operator.
+* You must update both the Elasticsearch Operator and the Cluster Logging Operator.
++
+Kibana is unusable when the Elasticsearch Operator has been updated but the Cluster Logging Operator has not been updated.
```

Comment:
That is not right now.   when you upgrade from 4.4 to 4.5. you needn't care about which one should be upgrade firstly. 

If you upgrade CLO first,  the fluentd upgrade will be blocked. You will see one fluentd pod in  Init:CrashLoopBackOff and show logs 'Elasticsearch is currently version: 5.6.16 - Expecting it to be at least: 6' .
$ oc logs -c fluentd-init fluentd-xp6xx
Elasticsearch is currently version: 5.6.16 - Expecting it to be at least: 6

Reply:
This is a case of "there may be dragons" and it doesn't cost us anything to have customers upgrade one before the other

Reply:
The pain point is if you forget to upgrade Logging from 4.4 to 4.5 in OCP 4.5.  After you upgrade to OCP 4.6.  No way to upgrade from 4.4 to 4.5 any more.  You have to upgrade from 4.4 to 4.6 directly or do a fresh install.

Reply:
@anpingli  Please advise us as to what the documentation should say so that we can resolve this comment. Thanks.